kalite/distributed/management/commands/setup.py

```diff
@@ -477,15 +477,10 @@ def handle(self, *args, **options):
         if not settings.CENTRAL_SERVER:
 
             kalite_executable = 'kalite'
-            if not spawn.find_executable('kalite'):
-                if os.name == 'posix':
-                    start_script_path = os.path.realpath(
-                        os.path.join(settings.PROJECT_PATH, "..", "bin", kalite_executable))
-                else:
-                    start_script_path = os.path.realpath(
-                        os.path.join(settings.PROJECT_PATH, "..", "bin", "windows", "kalite.bat"))
-            else:
```

Comment:
This part of the cleanup makes sense because `PROJECT_PATH` wasn't set to a directory containing the `bin/` directory, but rather then `~/.kalite` directory as we discontinued having the source checkout as user data directory.